### PR TITLE
Grammer Errors in Readme

### PR DESCRIPTION
**Pull request by anupamasok**
## Brief description
A few grammar errors and punctuation mistakes have been rectified.
...

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Further details

...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ After you make changes, be sure to build the new CSS files. Do so by running `gu
 
 ## Contributing
 
-This project is open source and contributions are very welcomed. It is also as beginner friendly as possible, so don't be afraid to jump in if you've never contributed to any Git project before! Feel free to reach out if you are new and need help with the process.
+This project is open source and contributions are very welcome. It is also as beginner friendly as possible, so don't be afraid to jump in if you've never contributed to any Git project before! Feel free to reach out if you are new and need help with the process.
```
Comment (TotomInc) [inline README.md]: `welcomed` and `welcome` are both correct in this case, but I prefer `welcomed` in this context.

File: README.md [hunk 2]
```diff
@@ -62,9 +62,9 @@ After you make changes, be sure to build the new CSS files. Do so by running `gu
...
 
-Please before sending a PR, make sure you are properly using the `.editorconfig` file with your IDE. If your IDE doesn't natively support `editorconfig` files, you can use an extension/package/module. For example in Atom there is the [editorconfig package](https://atom.io/packages/editorconfig), as well for [Sublime Text](https://github.com/sindresorhus/editorconfig-sublime), [VS Code](https://github.com/editorconfig/editorconfig-vscode), [Vim](https://github.com/editorconfig/editorconfig-vim), ...
+Please before sending a PR, make sure you are properly using the `.editorconfig` File with your IDE. If your IDE doesn't natively support `editorconfig` files, you can use an extension/package/module. For example in Atom there is the [editorconfig package](https://atom.io/packages/editorconfig), as well for [Sublime Text](https://github.com/sindresorhus/editorconfig-sublime), [VS Code](https://github.com/editorconfig/editorconfig-vscode), [Vim](https://github.com/editorconfig/editorconfig-vim), ...
```
Comment (TotomInc) [inline README.md]: Keep `file` in lowercase.

File: README.md [hunk 3]
```diff
@@ -75,19 +75,19 @@ Once you are ready to contribute, here the workflow you should follow:
...
+- Open a pull request on the origin papercss repo. Be sure to include any pictures and/or details on what you have done; it will help reviewers **a lot**!
+- When your changes are approved, they will be merged into the `develop` branch, which will finally be merged into the `master` branch when we reach a milestone regarding features and bug fixes. Check out [Vincent Driessen's blog post](http://nvie.com/posts/a-successful-git-branching-model/), [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html), or [#27](https://github.com/rhyneav/papercss/issues/27) for more details on how this works.
 
-Note: If you have a hotfix (usually typos and minor documentation tweaks), create your hotfix branch off of the master branch instead of develop: `git checkout -b hotfix-thing master`. The changes will be merged into both master and develop to keep the branches consistent.
+Note: If you have a hotfix (usually typos and minor documentation tweaks), create your hotfix branch off of the master branch instead of developing: `git checkout -b hotfix-thing master`. The changes will be merged into both the master and develop to keep the branches consistent.
```
Comment (TotomInc) [inline README.md]: If I'm correct we are talking about the `develop` branch, not the term itself. Let's keep `develop` instead of `developing`.

**Comment by rhyneav** [comment]
This is great, thank you for the updates!

**Pull request merged**